lib/jellyfish_web/api_spec/health_report.ex

```diff
@@ -0,0 +1,29 @@
+defmodule JellyfishWeb.ApiSpec.HealthReport do
+  @moduledoc false
+
+  require OpenApiSpex
+
+  defmodule Status do
+    @moduledoc false
+
+    require OpenApiSpex
+
+    OpenApiSpex.schema(%{
+      title: "HealthReportStatus",
+      description: "Informs about the status of Jellyfish",
+      type: :string,
+      enum: ["UP", "DOWN"],
```

Comment:
Fun fact: when adding string enum, we have to ensuer that the values aren't any of:
`'0', 'off', 'f', 'false', 'n', 'no', '1', 'on', 't', 'true', 'y', 'yes'` , since they will be converted to boolean when generating the python SDK. https://docs.pydantic.dev/2.0/usage/types/booleans/

In this case we are perfectly fine.

Reply:
I had no idea -- perhaps we should leave it as a memo somewhere?

Reply:
We may, however I am not sure where